js/private/js_image_layer.bzl

```diff
@@ -607,6 +607,7 @@ js_image_layer_lib = struct(
         ),
         "root": attr.string(
             doc = "Path where the files from js_binary will reside in. eg: /apps/app1 or /app",
+            default = "/",
```

Comment:
```suggestion
```

we should probably revert this line even though result will be the same. just to adhere to semver